serviceUsage.html

```diff
@@ -51,13 +51,27 @@ <h2>
             </a>Configuration
         </h2>
 
-        <p>For using Eiffel REMReM Service <strong>publish-service.war</strong> file should be deployed in Tomcat Server.
+        <p>For using Eiffel REMReM Service <strong>publish-service.war</strong> file is preferably deployed in Tomcat Server.
```

Comment:
Replace this line with a header Running and configuring in Tomcat.
Move the hole section for Tomcat after the section with instructions without Tomcat

Reply:
Fixed.